aesm-client/src/unix/mod.rs

```diff
@@ -0,0 +1,13 @@
+use std::path::Path;
+use imp::AesmClient;
+pub trait AesmClientExt {
+    fn with_path<P: AsRef<Path>>(path: P) -> Self;
+}
+
+impl AesmClientExt for crate::AesmClient {
```

Comment:
If you put this in `unix_aesm_client.rs`, you don't need to add a separate `AesmClient::with_path` function. Also, then this file has become so small, you can inline it in `lib.rs`